src/main/java/org/pesmypetcare/webservice/thirdpartyservices/adapters/firestore/FirestoreCollectionAdapter.java

```diff
@@ -288,7 +288,7 @@ private void checkArguments(Object[] fieldsAndValues) {
             if (fieldPath == null || fieldPath.toString().isEmpty()) {
                 throw new IllegalArgumentException(INVALID_FIELD_MESSAGE);
             }
-            query.whereArrayContains(fieldPath, value);
+            query = query.whereArrayContains(fieldPath, value);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Assignment of parameter 'query' is not allowed.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Webservice/pullRequest?prid=5633182)